server/api/servarr/base.ts

```diff
@@ -61,6 +77,7 @@ interface QueueItem {
   protocol: string;
   downloadClient: string;
   indexer: string;
+  episode: EpisodeResult;
```

Comment:
`episode` is not returned for Radarr so we should make this not required or extend the type just for the Sonarr response.